app/config/HBaseRestRepositoryConfigLoader.scala

```diff
@@ -17,11 +17,14 @@ object HBaseRestRepositoryConfigLoader extends HBaseRestConfigLoader[HBaseRestRe
   override def load(rootConfig: Config, path: String): HBaseRestRepositoryConfig = {
     val config = rootConfig.getConfig(path)
     HBaseRestRepositoryConfig(
-      baseUrl = BaseUrl(config.getString("protocol"), config.getString("host"), config.getInt("port"), Option(config.getString("prefix"))),
+      baseUrl = BaseUrl(config.getString("protocol"), config.getString("host"), config.getInt("port"), NoneEmptyString(config.getString("prefix"))),
       namespace = config.getString("namespace"),
       username = config.getString("username"),
       password = config.getString("password"),
       timeout = config.getLong("timeout")
     )
   }
-}
\ No newline at end of file
+  // Convert to None if string is empty or null
+  private def NoneEmptyString(x: String): Option[String] = if (x.trim.isEmpty) None else Some(x)
```

Comment:
Scala convention is for functions / methods to start with a lowercase letter: noneEmptyString.
x probably isn't the best parameter name - although I do often use it for simplicity when I'm just experimenting with something.  As the argument is a string, s or str are common names.